dotCMS/src/main/java/com/dotcms/rest/param/ISODateParam.java

```diff
@@ -0,0 +1,18 @@
+package com.dotcms.rest.param;
+
+import com.dotmarketing.util.DateUtil;
+
+import java.text.ParseException;
+import java.util.Date;
+
+/**
+ * Encapsulates the logic to parse a ISO String date into a {@link Date}
+ * @author jsanca
+ */
+public class ISODateParam extends Date {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Classes implementing Serializable should set a serialVersionUID](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=4502981)